riscv-page-tables/src/page_table.rs

```diff
@@ -727,10 +727,10 @@ impl<'a, T: FirstStagePagingMode> FirstStageMapper<'a, T> {
         page_size: PageSize,
```

Comment:
Please update the commit message. The subject is currently the same as the first patch; something like "SequentialPages: Add huge page tests" would be more appropraite.